pyproject.toml

```diff
@@ -36,12 +36,22 @@ emoji = [
     "emoji",
     "langchain-aws"
 ]
-all = ["jupyter-ai-personas[finance,emoji]"]
+
+pr_review = [
+    "agno",
+    "boto3",
+    "langchain",
+    "langchain-core",
```

Comment:
Don't think you need these. Jupyter AI depends on `langchain`, which should pull in `langchain-core`.